evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -187,3 +187,8 @@ def hours_and_minutes(time_left_for_evaluation):
 @register.filter
 def has_nonresponsible_editor(evaluation):
     return evaluation.contributions.filter(can_edit=True).exclude(contributor__in=evaluation.course.responsibles.all()).exists()
+
+
+@register.filter
+def order_by(evaluations, attribute):
+    return sorted(evaluations, key=lambda evaluation: getattr(evaluation, attribute))
```

Comment:
Not a big thing, but may be nice to have: The method is quite generic, you can not only order evaluations with it, but anything iterable, so I'd suggest renaming `evaluations` to `iterable` here.